experiments/README.md

```diff
@@ -40,11 +40,11 @@ type Experiment interface {
 	// Technique returns the attack method
```

Comment:
You should also update the table above in this file to include your experiment, to show users what's available